### PR TITLE
New Google Analytics tags

### PR DESCRIPTION
**Pull request by sidane**
Replace existing, old google analytics tags with the newer tags.

Only rendered when a google analytics account ID is set in an environment variable.

There is no longer a requirement to track user type, lender name and specific events (e.g. downloading a CSV).

### CODE REVIEW

File: app/views/shared/_google_analytics.html.erb [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
-<% if lender_name %>
-  _gaq.push(["_setCustomVar", 2, "Lender name", "<%= lender_name %>", 3]);
-<% end %>
-  _gaq.push(['_trackPageview']);
+<% if ENV["GOOGLE_ANALYTICS_ACCOUNT_ID"] %>
+  <script>
+    (function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,"script","https://www.google-analytics.com/analytics.js","ga");
```
Comment (nickcharlton) [inline app/views/shared/_google_analytics.html.erb]: Oh wow this is nasty. Good job Google.

**Comment by nickcharlton** [comment]
I like it. Much nicer.

File: app/views/shared/_google_analytics.html.erb (hunk 1 above)
Comment (sidane) [inline app/views/shared/_google_analytics.html.erb]: 😂

**Comment by sidane** [comment]
Thanks @nickcharlton

**Pull request merged**